kairos-deposit-contract/deposit-session/src/main.rs

```diff
@@ -0,0 +1,24 @@
+#![no_std]
+#![no_main]
+use casper_contract::contract_api::{account, runtime, system};
+use casper_types::{runtime_args, AddressableEntityHash, URef, U512};
+
+#[no_mangle]
+pub extern "C" fn call() {
+    let contract_hash: AddressableEntityHash = runtime::get_named_arg("deposit_contract");
+    let amount: U512 = runtime::get_named_arg("amount");
+    let source: URef = account::get_main_purse();
+    // create a temporary purse that can be passed to the deposit contract
+    let temp_purse: URef = system::create_purse();
+    // fund the temporary purse
+    system::transfer_from_purse_to_purse(source, temp_purse, amount, None).unwrap();
+    // call the deposit endpoint
```

Comment:
the code literally says `call_contract(..."deposit")`

Reply:
The entry point name is static, I can introduce a constant if you prefer that. Something like DEPOSIT_EP_NAME = "deposit".

Reply:
I haven't been clear enough with my comment: I mean documentation should be used to point out non-obvious things. This comment is repeating the code.

Reply:
Wouldn't it be easier to suggest a change directly than open a conversation in cases where you only want a single line to be removed or edited?